src/definition/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rocket.chat/apps-ts-definition",
-  "version": "1.5.2",
+  "version": "1.6.0-alpha",
```

Comment:
Two different versions in the various packages.

Reply:
@d-gubert Do you know why it happened?

Reply:
This is related to the `@rocket.chat/apps-ts-definition` package, for which we should drop updating. In fact, we haven't been updating it with the latest releases of the Engine.  We should remove that in some upcoming version, but I guess we could leave it there for now and deal with that in the beta branch